docs/release-notes/release-notes-0.18.0.md

```diff
@@ -278,6 +278,10 @@
   and makes TLV Onions, Static Remote Keys, Gossip Queries, compulsory features for
   LND's peers. Data Loss Protection has been compulsory for years.
 
+* [Implement feature bits and message in the peer backup proposal](https://github.com/lightningnetwork/lnd/pull/8490)
+  This PR implements the feature bits and messages in the peer backup proposal
+   referenced here: https://github.com/lightning/bolts/pull/1110
```

Comment:
As a matter of policy, you should never add the advertisement bits until the entire proposal is implemented properly. It should be the last step in the process. Any change you implement should always be done "back to front". Like so:

1. Implement core axioms/operations
2. Plug it into the rest of the system
3. Implement the on switch
4. Advertise